docker/conf/stack/docker-stack-1.yml

```diff
@@ -489,6 +489,8 @@ services:
       KC_JEMPI_ROOT_URL: ${KC_JEMPI_ROOT_URL}
       JEMPI_SESSION_SECRET: ${JEMPI_SESSION_SECRET}
       JEMPI_FILE_IMPORT_MAX_SIZE_BYTE: ${JEMPI_FILE_IMPORT_MAX_SIZE_BYTE}
+      JEMPI_SESSION_SECURE: ${JEMPI_SESSION_SECURE}
+      DOMAIN_NAME: ${DOMAIN_NAME}
```

Comment:
We need maybe to add a prefix "JEMPI_SESSION_"

Reply:
cool